MathsTest.java

```diff
@@ -0,0 +1,60 @@
+import static org.junit.Assert.assertEquals;
```

Comment:
Missing tests mentioned in assignment

rem(2, 1)
rem(2, 0)
rem(100, 3)

gcd(2, 1)
gcd(12, 18)
gcd(100, 3)

largestdigit(2)
largestdigit(1257369)
largestdigit(444)